vips/vip-xxx/bsctestnet.ts

```diff
@@ -0,0 +1,127 @@
+import { ProposalType } from "src/types";
+import { makeProposal } from "src/utils";
+
+export const NORMAL_TIMELOCK = "0xce10739590001705F7FF231611ba4A48B2820327";
+export const BNB_CORE_COMPTROLLER = "0x94d1820b2D1c7c7452A163983Dc888CEC546b77D";
+export const LIQUID_STAKING_BNB_COMPTROLLER = "0x596B11acAACF03217287939f88d63b51d3771704";
+export const DEFI_COMPTROLLER = "0x23a73971A6B9f6580c048B9CB188869B2A2aA2aD";
+export const STABLECOIN_COMPTROLLER = "0x10b57706AD2345e590c2eA4DC02faef0d9f5b08B";
+export const GAMEFI_COMPTROLLER = "0x1F4f0989C51f12DAcacD4025018176711f3Bf289";
+export const BTC_COMPTROLLER = "0xfE87008bf29DeCACC09a75FaAc2d128367D46e7a";
+export const TRON_COMPTROLLER = "0x11537D023f489E4EF0C7157cc729C7B69CbE0c97";
+export const LIQUID_STAKING_ETH_COMPTROLLER = "0xC7859B809Ed5A2e98659ab5427D5B69e706aE26b";
+export const MEME_COMPTROLLER = "0x92e8E3C202093A495e98C10f9fcaa5Abe288F74A";
+export const ACM = "0x45f8a08F534f34A97187626E05d4b6648Eeaa9AA";
+export const RISK_STEWARD_RECEIVER = "0x34e4505f92C8499B07eeb7Aa72404A490D152Ab3";
```

Comment:
I don't see this contract deployed at https://github.com/VenusProtocol/governance-contracts/pull/115